python/paddle/fluid/backward.py

```diff
@@ -614,6 +629,12 @@ def _addup_repetitive_outputs_(op_descs, block_idx, grad_var_to_var=None):
         # If not reverse, we first insert 'a' at idx 1, it becomes [0, 1, 'a', 2], and then insert 'b' at idx 2, it becomes [0, 1, 'a', 'b', 2].
         idx = key
         for i, op in enumerate(value):
+            # update the mapping between fwd and bwd
+            target_idx = idx - 1 if idx == op_descs_len else idx + i
```

Comment:
No need to update?

Reply:
Should update? These op descs are created from `_create_op_desc_` func. I think these updates are necessary.